e2e/test-runner/test_cases.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 // testUserNotRoot ensures the default user is not root.
-func testUserNotRoot(t *testing.T) {
+func testUserNotRoot(t *testing.T) { //nolint: thelper
```

Comment:
I would add an additional comment like ("This is a test"). Same below, but I won’t repeat it :)